### PR TITLE
feat: email section

### PR DESCRIPTION
**Pull request by simboonlong**
### User story
https://www.pivotaltracker.com/story/show/172118901

---

### PR updates
- email with netlify forms

---

email
netlify

### CODE REVIEW

File: src/components/HomePageContent/EmailForm.test.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { EmailForm } from "./EmailForm";
+
+describe("portal", () => {
```
Comment (Nebulis) [inline src/components/HomePageContent/EmailForm.test.tsx]: why portal ?

File: src/components/HomePageContent/EmailForm.test.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  it("should have hidden input value (contact) for netlify email to work", () => {
+    const container = render(<EmailForm />);
+
+    expect(container.getByDisplayValue("contact")).not.toBeNull();
```
Comment (Nebulis) [inline src/components/HomePageContent/EmailForm.test.tsx]: should test that the form is hidden ?

**Comment by Nebulis** [comment]
image

Changing this in another story ?

File: src/components/UI/Textarea/Textarea.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  name: string;
+  placeholder?: string;
+  required?: boolean;
+}
+
+export const Textarea = ({ ...props }: TextareaProps) => {
+  return <textarea {...props} />;
+};
+
+export const TextareaDefault = styled(Textarea)`
```
Comment (Nebulis) [inline src/components/UI/Textarea/Textarea.tsx]: you should be able to replace `styled(Textarea)` by `styled.textarea`

File: src/components/UI/Textarea/Textarea.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks";
+import { Textarea, TextareaDefault } from "./Textarea";
+
+<Meta title="UI/Textarea" component={Textarea} />
+
+# Textarea
+
+Textarea look.
```
Comment (Nebulis) [inline src/components/UI/Textarea/Textarea.stories.mdx:8]: Best description xD

File: src/components/UI/Input/Input.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  value?: string;
+  required?: boolean;
+  placeholder?: string;
+}
+
+export const Input = ({ ...props }: InputProps) => {
+  return <input {...props} />;
+};
+
+export const InputDefault = styled(Input)`
```
Comment (Nebulis) [inline src/components/UI/Input/Input.tsx]: styled.input

File: src/components/UI/Checkbox/Checkbox.tsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { InputDefault } from "../Input";
+
+interface CheckboxProps {
```
Comment (Nebulis) [inline src/components/UI/Checkbox/Checkbox.tsx]: can reuse InputHTMLAttributes eventually

File: src/components/UI/Textarea/Textarea.tsx (hunk 3 above)
Comment (simboonlong) [reply src/components/UI/Textarea/Textarea.tsx]: yup ok

**Comment by simboonlong** [comment]
> image
> 
> Changing this in another story ?

Yeah this will be separate story

**Review by Nebulis — APPROVED** [review]

File: src/components/UI/Checkbox/Checkbox.tsx (hunk 6 above)
Comment (simboonlong) [reply src/components/UI/Checkbox/Checkbox.tsx]: :thumbsup:

File: src/components/HomePageContent/EmailForm.test.tsx (hunk 2 above)
Comment (simboonlong) [reply src/components/HomePageContent/EmailForm.test.tsx]: ok updated

**Pull request merged**